src/styles/product-card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
+  border: 1px solid rgba(243,243,243,1);
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__price-amount {
+    display: flex;
+    font-size: 16px;
+    font-weight: 700;
+    color: $color_2;
+    margin-left: auto;
+    width: 50px;
+    height: 18px;
+    text-align: right;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    width: 166px;
+    height: 16px;
+    line-height: 18px;
+    margin: 0 auto 16px;
+    font-size: 12px;
+  }
+
+  &__apple {
+    font-size: 12px;
+    line-height: 18px;
+    margin: 40px auto 0;
+    width: 166px;
+    height: 36px;
+    font-weight: 500;
+    color: $color_2;
+  }
+
+  &__code {
+    font-weight: 400px;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 4px 16px 16px;
+    color: #616070;
+  }
+
+  &__reviews-text {
+    width: 53px;
+    height: 16px;
+    line-height: 14px;
+    color: rgba(6, 11, 53, 1);
+    font-size: 10px;
+    font-weight: 400;
+    text-align: center;
+    margin-top: 2px;
+    vertical-align: middle;
+  }
+}
+
+.stars--4 {
+  .stars_star {
+    &:nth-child(-n+4) {
+      background-image: url(/images/star-active.svg);
+      font-size: 12px;
+    }
+  }
+}
```

Comment:
You are styling stars here while you have a file for styles. Please do something about it ;)